src/Maker/MakeRegistrationForm.php

```diff
@@ -244,11 +304,53 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
             $this->fileManager->dumpFile($classDetails->getPath(), $userManipulator->getSourceCode());
         }
 
+        if ($input->getArgument('will-verify-email')) {
+            $classDetails = new ClassDetails($userClass);
+            $userManipulator = new ClassSourceManipulator(
+                file_get_contents($classDetails->getPath())
+            );
+            $userManipulator->setIo($io);
+
+            $userManipulator->addProperty('verified', ['@ORM\Column(type="boolean")'], false);
+            $userManipulator->addAccessorMethod('verified', 'isVerified', 'bool', false);
+            $userManipulator->addSetter('verified', 'bool', false);
+
+            $this->fileManager->dumpFile($classDetails->getPath(), $userManipulator->getSourceCode());
+        }
+
         $generator->writeChanges();
 
         $this->writeSuccessMessage($io);
-        $io->text('Next: Go to /register to check out your new form!');
-        $io->text('Make any changes you need to the form, controller & template.');
+        $this->successMessage($io, $input->getArgument('will-verify-email'), $userClassNameDetails->getShortName());
+    }
+
+    private function successMessage(ConsoleStyle $io, bool $emailVerification, string $userClass): void
+    {
+        $closing[] = 'Next:';
+
+        if (!$emailVerification) {
+            $closing[] = 'Make any changes you need to the form, controller & template.';
+        } else {
+            $closing[] = '1) Set the redirect route on successful email verification in the <fg=yellow>RegistrationController</>';
+            $closing[] = '2) Display the success flash message in your redirect template, or remove the message all together.';
+            $closing[] = '3) Review and customize the form, controller, and templates as needed.';
+            $closing[] = sprintf('4) Run <fg=yellow>"php bin/console make:migration"</> to generate migrations for the newly added <fg=yellow>%s::verified</> property.', $userClass);
+        }
+
+        $io->text($closing);
+        $io->newLine();
+        $io->text('Then open your browser, go to "/registration" and enjoy your new form!');
+        $io->newLine();
+    }
+
+    private function checkVerifyBundleExists(ConsoleStyle $io): void
+    {
+        if (!class_exists(SymfonyCastsVerifyEmailBundle::class)) {
+            $warning[] = 'Don\'t forget to install the Symfony Casts Verify Email Bundle after you\'re finished.';
```

Comment:
```suggestion
            $warning[] = 'After finishing this command, you will need to run:';
```

And how does the `->warning()` look? I'd like to make it kind obvious, but I don't want it to look like an error.

Also, we should mention this again in `successMessage()` so that it's at the bottom of the output.